azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -221,6 +222,23 @@ def task_any(self, activities: List[Task]) -> TaskSet:
         """
         raise NotImplementedError("This is a placeholder.")
 
+    def set_custom_status(self, status: Any):
```

Comment:
Is the status really `Any` or is it restricted to a JSON-serializable type? Perhaps Any makes sense for now, but I'm curious

Reply:
Its actually "Any" - a string or object in JSON language. 
So the JSON schema is anyOf {"type" : "string"} or { "type" : "object"}